EventFilter/GEMRawToDigi/python/muonGEMDigis_cfi.py

```diff
@@ -7,5 +7,8 @@
 from Configuration.Eras.Modifier_phase2_GEM_cff import phase2_GEM
 
 run2_GEM_2017.toModify(muonGEMDigis, useDBEMap = True)
-run3_GEM.toModify(muonGEMDigis, useDBEMap = True)
-phase2_GEM.toModify(muonGEMDigis, useDBEMap = False, readMultiBX = True)
+# from DataFormats/FEDRawData/interface/FEDNumbering.h
+MINGE21FEDID = 1469
+MAXGEMFEDID = 1478
```

Comment:
hardcoding these values here in addition to the header file could potentially be problematic. on the other hand, it seems that FEDIDs are commonly hardcoded in our python configs, and I can't suggest of a better way: it would either have to be propagated from the python config to FEDNumbering.h (meaning a reworking of the `FEDNumbering` class), or accessing the header information from python here somehow (which I don't think is possible).

pinging @jshlee in case you have any suggestions on this.

Reply:
We can add a phase/run parameter in GEMRawToDigiModule to choose the FEDIDs from FEDNumbering, which will remove the hard coding of these values.

Reply:
that could work too. phase2 flags are fairly common in modules.